src/main.cpp

```diff
@@ -15,13 +15,18 @@
 const int mainBufSize = 128;
 char mainBuf[mainBufSize];
 
+#if defined(CANBUS_ENABLED) && defined(BMS_TX_PIN) && defined(BMS_ON_PIN)
+  #include "BMSController.h"
+#endif
+
 unsigned long mainLoop = 0;
 unsigned long loopTime = 0;
 unsigned long maxLoopTime = 0;
 int new_forward = LOW;
 int new_backward = LOW;
 int new_brake = LOW;
 int idle = LOW;
+int mall_grab = LOW;
```

Comment:
Can you please help me to understand what this one is used for?

Reply:
This changes the idle pattern on the board to BATTERY_INDICATOR when pitch > 70deg. This is very useful for boards that don't have a separate LED Bar for battery status and still allows for cool idle pattern colors when the board is resting on the ground. :) Mall grabbing is a old skater term of carying the board by it's font truck (same position as the handle on a onewheel lol)